bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/TableJsonDataSource.java

```diff
@@ -88,7 +88,7 @@ public Object build() {
         BIterator itr = this.tableValue.getIterator();
         while (itr.hasNext()) {
             TupleValueImpl tupleValue = (TupleValueImpl) itr.next();
-            MapValueImpl record = ((MapValueImpl) tupleValue.get(0));
```

Comment:
Shall we make a comment saying what this index means?

Reply:
@mohanvive

Reply:
Added a comment for the opeartion.